101-backup-oms-monitoring/azurebackupmonitoringsolution.json

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
what's the purpose of this file vs. azuredeploy.json?

Reply:
I just added it in the beginning. Then realized that azuredeploy.json has the relevant data. azurebackupmonitoring.json can be removed.